tsconfig/tsconfig.esnext.cls.json

```diff
@@ -0,0 +1,12 @@
+// Compiler configuaration to build the intermediate closure-compiler ESM UMD target.
+// Not used at the moment, but ready whenever CC has full ES2015+ support.
```

Comment:
I just noticed these comments are out of date as of yesterday, these files _are_ being used to create the intermediate `cls` target that gets Ugly-fied